### ISSUE TITLE
Info Screen Crash

### ISSUE THREAD

**Issue by CaveJohnsonn** [FTBTeam/FTB-Continuum#378]
+Modpack Version: FTB - Continuum v1.4.1
+Issue: Worlds crash unexpectedly, crash log points to Information Screens.
+Log: https://pastebin.com/hh4LrkmM
+Is It Repeatable?: Yes, it repeats itself, I'm unsure of how or why, but it's nothing that I've done. It happens when standing still or moving, not interacting with anything.
+Mod/s affected: The Log points at RFTools, IE and a few others.
+Known Fix: N/A

**Comment by sergesountsov** [comment]
I am seeing exactly same issue. I think it might has something to do with RFTools Power cell and chunk loading/unloading. I had a crash happen first when I was flying on a JetPack and flew away from my base. After I reloaded I was able to get back to the base and one of the cells had One Probe information all screwed up. After picking it up with a wrench and placing it back I was able to read the charge status with One Probe clearly. 
For a while my world was not crashing and then I decided to go to the nether. And it crashed as soon as I entered the nether dimension.
